src/database/utilities/createRecord.js

```diff
@@ -93,36 +102,24 @@ const createPrescriber = (database, prescriberDetails) => {
   database.save('Prescriber', prescriber);
 };
 
-const createAddress = (database, { line1, line2, line3, line4, zipCode } = {}) =>
-  database.create('Address', {
-    id: generateUUID(),
-    line1,
-    line2,
-    line3,
-    line4,
-    zipCode,
-  });
-
 /**
  * Creates a patient record. Patient details passed can be in the shape:
  *  {
  *    firstName, lastName, dateOfBirth, code, emailAddress,
- *    phoneNumber, line1, line2,
+ *    phoneNumber, addressOne, addressTwo, country
```

Comment:
`createAddress` is still expecting a `line1` and `line2` and these are taken from the `patientDetails` prop
I would have thought:

```
  const { dateOfBirth, addressOne, addressTwo } = patientDetails;
  const billingAddress = createAddress(database, { line1: addressOne, line2: addressTwo });
```

Reply:
whoops! e579fa01da7667d92bc0f56ff6515e2219f7d260